test/functional/JIT_Test/playlist.xml

```diff
@@ -118,6 +118,53 @@
 		</impls>
 	</test>
 
+	<test>
+		<testCaseName>jit_jitt_openj9_SCCnone</testCaseName>
```

Comment:
Can you change it to "jit_jitt_openj9_none_SCC" ? The same to other test case names.